backend/clubs/migrations/0054_auto_20201101_1152.py

```diff
@@ -0,0 +1,55 @@
+# Generated by Django 3.1.2 on 2020-11-01 16:52
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("clubs", "0053_auto_20201031_1456"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="club",
+            name="recruiting_cycle",
+            field=models.IntegerField(
+                choices=[(1, "Unknown"), (2, "Fall"), (3, "Spring"), (4, "Both")], default=1
+            ),
+        ),
+        migrations.AddField(
+            model_name="historicalclub",
+            name="recruiting_cycle",
+            field=models.IntegerField(
+                choices=[(1, "Unknown"), (2, "Fall"), (3, "Spring"), (4, "Both")], default=1
+            ),
+        ),
+        migrations.AlterField(
+            model_name="club",
+            name="application_required",
+            field=models.IntegerField(
+                choices=[
+                    (1, "Open Membership"),
+                    (3, "Audition Required"),
+                    (2, "Tryout Required"),
+                    (4, "Application Required"),
+                    (5, "Application and Interview Required"),
+                ],
+                default=4,
+            ),
+        ),
+        migrations.AlterField(
```

Comment:
These entries shouldn't appear in this commit. Perhaps you need to delete this commit, rebase with master, and regenerate this commit?

Reply:
Ah, never mind, I see that this is a typo fix. Nice!

Reply:
I changed a type in "Intereview" to "Interview".

Reply:
Is it still unintended?